### PR TITLE
Add type annotations to openapi.yml

### PR DESCRIPTION
**Pull request by aleksej-paschenko**

### CODE REVIEW

File: api/openapi.yml [hunk 1]
```diff
@@ -3582,6 +3598,7 @@ components:
           example: "1219de582369ac80ee1afe12147930f458a54ff1eea612611a8bc6bd31581a6c"
         raw_body:
           type: string
+          format: cell
           description: hex-encoded BoC with raw message body
           example: "B5EE9C7201010101001100001D00048656C6C6F2C20776F726C64218"
```
Comment (mois-ilya) [inline api/openapi.yml:3603]: Maybe change to lower case?

File: api/openapi.yml [hunk 2]
```diff
@@ -4534,9 +4571,10 @@ components:
             - tuple
         cell:
           type: string
-          example: "te6cckEBAQEAJAAAQ4ARPeUceMlv4l12d6jdLpIzzbAV6amYXNZeZK2aicQdC/Apj8aJ"
+          format: cell-base64
```
Comment (mois-ilya) [inline api/openapi.yml:4574]: Received hex
```curl
curl --location 'https://tonapi.io/v2/blockchain/accounts/EQDW6q4sRqQwNCmW4qwUpeFSU1Xhd6l3xwJ6jjknBPzxKNtT/methods/royalty_params'
```

Comment (aleksej-paschenko) [reply api/openapi.yml]: fixed, now it is `format: cell`

File: api/openapi.yml [hunk 3]
```diff
@@ -6796,6 +6873,7 @@ components:
             properties:
               address:
                 type: string
+                format: address
                 example: "0:010cеeac44fad23417a5c55e4071796868771082с9a61e8c195a8d57508b8471"
```
[the remark this answers is not in the document]
Comment (aleksej-paschenko) [reply api/openapi.yml:6877]: fixed

File: api/openapi.yml [hunk 4]
```diff
@@ -4534,9 +4571,10 @@ components:
             - tuple
         cell:
           type: string
-          example: "te6cckEBAQEAJAAAQ4ARPeUceMlv4l12d6jdLpIzzbAV6amYXNZeZK2aicQdC/Apj8aJ"
+          format: cell-base64
         slice:
           type: string
+          format: cell
           example: ""
```
[the remark this answers is not in the document]
Comment (aleksej-paschenko) [reply api/openapi.yml:4578]: fixed

**Pull request merged**